rcl/src/rcl/logging.c

```diff
@@ -88,15 +88,11 @@ rcl_logging_configure_with_output_handler(
   if (g_rcl_logging_ext_lib_enabled) {
     status = rcl_logging_external_initialize(config_file, g_logging_allocator);
     if (RCL_RET_OK == status) {
-      // TODO(dirk-thomas) the return value should be typed and compared to
-      // constants instead of zero
-      int logging_status = rcl_logging_external_set_logger_level(
-        NULL, default_level);
-      if (logging_status != 0) {
-        status = RCL_RET_ERROR;
+      status = rcl_logging_external_set_logger_level(NULL, default_level);
+      if (RCL_RET_OK == status) {
```

Comment:
This changes a logic, only when log level can be set, then register handler?
Previously registering handler anyway even if log level cannot be set.

Reply:
Yes. And I think this is the correct logic.

Reply:
@fujitatomoya 
After consideration, I shouldn't update the original logic while refactoring the source code.
I will update it.

Reply:
@iuhilnehc-ynos

I guess that original logic is meant to, I'd like to hear from others.